src/modules/app.js

```diff
@@ -78,20 +98,52 @@ class ToDoApp {
     textTask.addEventListener('focusout', () => {
       setTimeout(() => {
         this.setClasses(false, li, ellipsisIcon, trashIcon, 'move');
-      }, 120);
+      }, 220);
+    });
+  }
+
+  addCheckBoxListener = ($checkBox, $textTask, $square, $check, index) => {
+    $checkBox.addEventListener('change', () => {
+      if ($checkBox.checked) {
+        this.completedTaskArr.push(index);
+      } else {
+        this.completedTaskArr = this.completedTaskArr.filter((el) => el !== index);
+      }
+      $check.classList.toggle('hidden');
+      $square.classList.toggle('hidden');
+      $textTask.classList.toggle('underlined');
+      this.taskArr[index].completed = !this.taskArr[index].completed;
+      this.saveLocalStorage();
+      this.checkCompletedArr();
```

Comment:
- [ ] The status of the task should be updated when the checkbox is clicked and the status should also be stored in the local storage. Currently, the status changes back to false when refreshed, kindly modify that.

Reply:
## Hi @ZeenatLawal 👋,
Thanks for finding the bug, I'll fix it.

Reply:
@andres-condezo You are welcome. Best of luck 👍🏼

Reply:
Well addressed this!